README.md

```diff
@@ -78,6 +92,37 @@ For information about contributing to this project please refer to this
 * Releasing new versions
 * Docker development
 
+## CI/Testing Strategy
+
+The following parameters are considered in the testing strategy
+in the CI for this project:
+### Database
+
+We are testing this starter kit against PostgreSQL and MySQL.
```

Comment:
Thoughts on adding SQLite to test development? We're doing that on the main Solidus and at least some extensions.